content/en/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -9,37 +9,37 @@ weight: 30
 
 <!-- overview -->
 
-This page provides a real world example of how to configure Redis using a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. 
+After you [configure a Pod to use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/), you can then use a ConfigMap to configure Redis.
 
+## What you'll learn
 
-
-## {{% heading "objectives" %}}
-
+In this tutorial, you'll learn how to do the following tasks:
 
 * Create a ConfigMap with Redis configuration values
 * Create a Redis Pod that mounts and uses the created ConfigMap
 * Verify that the configuration was correctly applied.
 
+## Requirements
 
+- Kubernetes cluster
+- kubectl command-line tool (`kubectl` 1.14 and above) configured to communicate with your cluster.
```

Comment:
I think this statement is misleading. It implies you could use kubectl 1.14, but that is not correct for most users today. A specific version reference could probably be dropped here since 1.14 (and many more of the later ones) are well end of life by this point.